include/experimental/__p2630_bits/submdspan_mapping.hpp

```diff
@@ -0,0 +1,157 @@
+
+namespace std {
+namespace experimental {
+
+//******************************************
+// Return type of submdspan_mapping overloads
+//******************************************
+template <class Mapping> struct mapping_offset {
+  Mapping map;
```

Comment:
Would you consider calling this `mapping` instead of `map`, to avoid collision with `std::map`?
```suggestion
  Mapping mapping;
```

Reply:
yeah should do this.